services-core/gradle.properties

```diff
@@ -1,5 +1,5 @@
 VERSION_NAME=4.9.0-SNAPSHOT
 POM_ARTIFACT_ID=mapbox-sdk-core
 POM_NAME=Mapbox Services SDK
```

Comment:
This seems unused.

Reply:
Removed `POM_NAME` from each `gradle.properties` file